plugins/AudioFileProcessor/AudioFileProcessor.cpp

```diff
@@ -428,9 +428,9 @@ void AudioFileProcessor::loopPointChanged( void )
 
 void AudioFileProcessor::pointChanged( void )
 {
-	const f_cnt_t f_start = static_cast<f_cnt_t>( m_startPointModel.value() *	( m_sampleBuffer.frames()-1 ) );
-	const f_cnt_t f_end = static_cast<f_cnt_t>( m_endPointModel.value() * ( m_sampleBuffer.frames()-1 ) );
-	const f_cnt_t f_loop = static_cast<f_cnt_t>( m_loopPointModel.value() * ( m_sampleBuffer.frames()-1 ) );
+	const f_cnt_t f_start = static_cast<f_cnt_t>( m_startPointModel.value() *	( m_sampleBuffer.frames() ) );
+	const f_cnt_t f_end = static_cast<f_cnt_t>( m_endPointModel.value() * ( m_sampleBuffer.frames() ) );
+	const f_cnt_t f_loop = static_cast<f_cnt_t>( m_loopPointModel.value() * ( m_sampleBuffer.frames() ) );
```

Comment:
```suggestion
	const f_cnt_t f_start = static_cast<f_cnt_t>(m_startPointModel.value() * m_sampleBuffer.frames());
	const f_cnt_t f_end = static_cast<f_cnt_t>(m_endPointModel.value() * m_sampleBuffer.frames());
	const f_cnt_t f_loop = static_cast<f_cnt_t>(m_loopPointModel.value() * m_sampleBuffer.frames());
```

Reply:
To minimize my style changes, I'll remove the redundant paraentheses but keep the extra spaces and tab. Is this okay?

Reply:
I think you don't have to minimize style changes here because they are already in the line you're touching, but I will leave the decision up to you.